code/src/nuvla/connector/utils.py

```diff
@@ -48,6 +48,14 @@ def append_os_env(env):
     return final_env
 
 
+class CMDExecutionException(Exception):
+    ...
+
+
+class CMDTimeOutException(Exception):
```

Comment:
Where this exception is raised ?

Reply:
it is docker-compose.py (Line 288.), DockerCompose class, config method (I don't know how to reference the code in comments).

Reply:
Here https://github.com/nuvla/job-engine/blob/docker-compose-validator-bugfix/code/src/nuvla/connector/docker_compose.py#L288 ?

Reply:
Yes, I created those exceptions to prevent having to catch a wide Exception clause

Reply:
On the function `config` the exception is used in a `except` block but I don't see in the code where this exception is raised.

Reply:
This is the exception `CMDExecutionException`. I was asking for `CMDTimeOutException`.

Reply:
Oh, yes. You are right. I believe that I wrote it to raise it instead of the wide Exception in the same function in the TimeOut except. Should I delete it?

Reply:
You can raise it instead of Exception here: 
https://github.com/nuvla/job-engine/blob/952485444962a04db4d232731fb2921fce28c701/code/src/nuvla/connector/utils.py#L75